src/main/java/org/example/Cinema.java

```diff
@@ -0,0 +1,74 @@
+package org.example;
+
+public class Cinema {
+    private int row = 5;
+    private int place = 10;
+    private int[][] cinema = new int[row][place];
+
+    public Cinema() {
+        for(int i = 0; i < cinema.length; i++){
+            for (int j = 0; j < cinema[i].length; j++){
+                cinema[i][j] = j + 1;
+            }
+        }
+    }
+
+    public Cinema(int[][] cinema) {
+        this.cinema = cinema;
+    }
+
+    public int[][] getCinema() {
+        return cinema;
+    }
+
+    public void setCinema(int[][] cinema) {
+        this.cinema = cinema;
+    }
+
+    public void printCinema() {
```

Comment:
jak zmienisz koncepcję zgodnie z moimi wskazówkami to ta metoda nie będzie potrzebna